pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -130,7 +131,38 @@ func (cl *Client) Run(stop <-chan struct{}) {
 			return
 		}
 		scope.Info("Pilot K8S CRD controller synced ", time.Since(t0))
-		cl.queue.Run(stop)
+		for kind, q := range cl.queueMap {
+			handler, ok := cl.kinds[kind]
+			if !ok {
+				continue
+			}
+			scope.Infof("start go kind: %s queueMap len:%d", kind, len(cl.queueMap))
+			for i := 0; i < 2; i++ {
```

Comment:
Maybe make this configurable

Reply:
Why do we need multiple workers?

Reply:
@howardjohn This is to make it work faster, just like kube-controller-manager does.